modules/calc/src/main/java/com/opengamma/strata/calc/Column.java

```diff
@@ -34,66 +36,122 @@
 @BeanDefinition
 public final class Column implements ImmutableBean {
 
-  /** The definition of the column which specifies the column name and the measures it contains. */
+  /**
+   * The measure to be calculated.
+   * <p>
+   * This defines the calculation being performed, such as 'PresentValue' or 'ParRate'.
+   */
   @PropertyDefinition(validate = "notNull")
-  private final ColumnDefinition definition;
-
-  /** The pricing rules that apply to this column in addition to the default rules. */
+  private final Measure measure;
+  /**
+   * The column name.
+   * <p>
+   * This is the name of the column, and should be unique in a list of columns.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final ColumnName name;
+  /**
+   * The pricing rules that apply to this column, overriding the default rules.
+   * In most cases, there is no need to override the rules on a per-column basis.
+   */
   @PropertyDefinition(validate = "notNull")
   private final PricingRules pricingRules;
-
-  /** The market data rules that apply to this column in addition to the default rules. */
+  /**
+   * The market data rules that apply to this column, overriding the default rules.
+   * In most cases, there is no need to override the rules on a per-column basis.
+   */
   @PropertyDefinition(validate = "notNull")
   private final MarketDataRules marketDataRules;
+  /**
+   * The reporting currency that applies to this column, overriding the default reporting currency.
+   * The common case is to specify a default reporting currency
+   */
+  @PropertyDefinition(get = "optional")
+  private final ReportingCurrency reportingCurrency;
 
-  /** The reporting rules that apply to this column in addition to the default rules. */
-  @PropertyDefinition(validate = "notNull")
-  private final ReportingRules reportingRules;
-
-  @ImmutableDefaults
-  private static void applyDefaults(Builder builder) {
-    builder.pricingRules(PricingRules.empty());
-    builder.marketDataRules(MarketDataRules.empty());
-    builder.reportingRules(ReportingRules.empty());
+  //-------------------------------------------------------------------------
+  /**
+   * Obtains an instance from the specified measure.
```

Comment:
"from the specified measure" painted an inaccurate picture in my mind of what was happening. As if the measure was somehow a factory for columns.

A column contains values of a measure.